packages/frontend/src/assets/locales/fi/common.json

```diff
@@ -29,6 +29,7 @@
     "description": "Selitysteksti",
     "more_actions": "Valinnaiset toiminnot",
     "file_not_selected": "Tiedostoa ei valittu",
-    "file_not_uploaded": "Tiedostoa ei ladattu"
+    "file_not_uploaded": "Tiedostoa ei ladattu",
```

Comment:
Should uploading be translated as "lähettäminen"? To avoid confusion. Just a suggestion.

Reply:
That's right! Good point 👍🏻